server/app/views/questiontypes/ApplicantQuestionRendererImpl.java

```diff
@@ -48,16 +59,22 @@ private String getRequiredClass() {
 
   protected abstract DivTag renderTag(
       ApplicantQuestionRendererParams params,
-      ImmutableMap<Path, ImmutableSet<ValidationErrorMessage>> validationErrors);
+      ImmutableMap<Path, ImmutableSet<ValidationErrorMessage>> validationErrors,
+      ImmutableList<String> ariaDescribedByIds,
+      boolean hasErrors);
 
   @Override
   public final DivTag render(ApplicantQuestionRendererParams params) {
+    boolean hasErrors = false;
+    ImmutableList.Builder<String> ariaDescribedByBuilder = ImmutableList.builder();
```

Comment:
Not blocking: You can chain the initial addition:

```
ImmutableList.Builder<String> ariaDescribedByBuilder = ImmutableList.builder()
  .add(descriptionId);
```

Reply:
I think this can be removed by also adding the type annotation to the builder method (e.g.):

```
ImmutableList.Builder<String> ariaDescribedByBuilder = ImmutableList.<String>builder()
  .add(...);
```

Reply:
Thank you! Done.